ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -11,6 +11,72 @@
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
+# global
+from hypothesis import given, strategies as st
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+import ivy
+
+# nanmedian
+@handle_cmd_line_args
```

Comment:
you should not use this old way to write frontend test, please check the contribution guide and use something like this
@handle_frontend_test(
    fn_tree="numpy.mean",
    dtype_and_x=statistical_dtype_values(function="mean"),
    dtype=helpers.get_dtypes("float", full=False, none=True),
    where=np_frontend_helpers.where(),
    keep_dims=st.booleans(),
)
def test_numpy_mean(
    dtype_and_x,
    dtype,
    where,
    frontend,
    test_flags,
    fn_tree,
    on_device,
    keep_dims,
):

Reply:
Done

Reply:
you no longer need the "@handle_cmd_line_args" and "@given". Please, check out this guide(https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#ivy-frontend-tests)